apps/channels/tests/test_base_channel_behavior.py

```diff
@@ -29,7 +29,7 @@
 def telegram_channel(db):
     experiment = ExperimentFactory(conversational_consent_enabled=False)
     channel = ExperimentChannelFactory(experiment=experiment)
-    channel = TelegramChannel(experiment, channel)
+    channel = TelegramChannel(experiment=experiment, experiment_channel=channel)
```

Comment:
just curious by why the change back to kwargs?

Reply:
It's just preference. I get somewhat "twitchy" whenever there's multiple args being passed to a function that relies on position.